lib/rouge/lexers/email.rb

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class Email < RegexLexer
+      tag 'email'
+      aliases 'eml', 'e-mail'
+      filenames '*.eml'
+      mimetypes 'message/rfc822'
+
+      title "Email"
+      desc "An email message"
+
```

Comment:
@smokris You had defined a `detect?` method and used the heuristic of whether the e-mail began with `From:`. Is this necessary? It's not a requirement for lexers to define this method and I'd generally recommend against it unless there's a pressing reason to the contrary. What were your thoughts on this?

Reply:
I was thinking that, in the context of triple-backtick code fences in Markdown, it would be helpful to automatically detect an email if the text author didn't explicitly specify it by appending `email` to the opening triple-backtick.  But since there isn't a 100%-confidence way to detect that a block of text is an email (no shebang or doctype), I'm OK with omitting the detector.